### ISSUE TITLE
Extendings Tags:  File is invalid: Unknown "cache" tag.

### ISSUE THREAD

**Issue by timvonboard** [VincentLanglet/Twig-CS-Fixer#61]
Hey there, 
is it possible to add tags? or exclude single files?

I try to fix this line: 
```
{% cache 'cache_key' ttl(300) %}
    say hello cache
{% endcache %}
```
And i´m getting: 
` File is invalid: Unknown "cache" tag.`

**Comment by VincentLanglet** [comment]
How do you create a new tag ?
Can you confirm you have a specific token parser

It should be able to support it.
The issue is with the parser of the file 
https://github.com/VincentLanglet/Twig-CS-Fixer/blob/13b660fa9d9764345a0d181a6c898da5318fb604/src/Runner/Linter.php#L113-L121

And I use a stubbed environnement for this https://github.com/VincentLanglet/Twig-CS-Fixer/blob/13b660fa9d9764345a0d181a6c898da5318fb604/src/Environment/StubbedEnvironment.php#L23

So we need to add your token parser to this environnment.

The stub environement is created here: https://github.com/VincentLanglet/Twig-CS-Fixer/blob/main/src/Command/TwigCsFixerCommand.php#L67

We could add a config to add custome token parser.

**Comment by timvonboard** [comment]
Sry, dont have a specific parser.
I just have Problems with this one:

https://twig.symfony.com/doc/3.x/tags/cache.html
`{% cache "cache key" ttl(300) %}
    Cached for 300 seconds
{% endcache %} `

**Comment by VincentLanglet** [comment]
Oh, I thought this was a custom one. It's even easier then.

Can you try this PR https://github.com/VincentLanglet/Twig-CS-Fixer/pull/63 ?

**Comment by timvonboard** [comment]
Check, works perfect.
` [OK] Files linted: 1, notices: 0, warnings: 0, errors: 0`

Thank you for your amazingly fast response!

**Comment by VincentLanglet** [comment]
Released as 0.5.1
